index.html

```diff
@@ -38,6 +38,7 @@
       data-cfasync="false"
       data-ui="http://localhost:8080/build/ui.js"
       src="https://cdn.transcend.io/cm/443312ef-12f9-494d-85f5-969894260cc7/airgap.js"
+      data-regime="GDPR"
```

Comment:
Looks like this was left in from testing

```suggestion
```

Reply:
thought i would leave it in since it makes the UI a bit more rich. may be nice to add buttons back like we had when we first built this

Reply:
oh that's fine then